truss/cli.py

```diff
@@ -297,43 +295,24 @@ def train(target_directory: str, build_dir, tag, var: List[str], vars_yaml_file,
 
 @cli_group.command()
 @click.argument("target_directory", required=False, default=os.getcwd())
-@click.option("--api-key", type=str, required=False, help="Your API key")
+@click.option("--service-name", type=str, required=True, help="Name of the service")
```

Comment:
can we make this not required in case that they only have one remote in their rc and we can just automatically use that or prompt them to choose if they have more than one?
This can be done in a followup if needed.